CHANGELOG.md

```diff
@@ -19,8 +19,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Add PARL_IO TX driver for ESP32-C6 / ESP32-H2 (#733)
 - Implement `ufmt_write::uWrite` trait for USB Serial JTAG (#751)
 - Add HMAC peripheral support (#755)
-- Add multicore-aware embassy executor for Xtensa MCUs (#723).
-- Add interrupt-executor for Xtensa MCUs (#723).
```

Comment:
Why are those CHANGELOG entries removed?

Reply:
they were duplicates caused by incorrect rebasing

Reply:
Duplicate entry, to avoid needing to rebase PRs all the time we modify the merge settings for that file in `.gitattributes`, it's not perfect so sometimes things need cleaning up.

Reply:
Gotcha. I didn't see they were present 2 lines below :sweat_smile: